melos.yaml

```
@@ -22,3 +22,6 @@ scripts:
   format-check: dart format --output=none --set-exit-if-changed --line-length 120 .
   analyze: dart analyze --fatal-infos .
   test: melos exec --concurrency=1 --fail-fast --dir-exists=test -- "flutter test"
+  generate_neon: melos exec --scope="neon*" -- dart run build_runner build --delete-conflicting-outputs
+  format: melos exec dart format --fix --line-length 120 .
```

```suggestion
  format: dart format --fix --line-length 120 .
```

You don't need to run it for every package individually. See the `format-check` script.